src/hooks/polymesh/useHistoricData.tsx

```diff
@@ -31,8 +32,11 @@ const useHistoricData = ({ pageIndex, pageSize }: IPaginationState) => {
       try {
         setDataLoading(true);
 
-        const { data, count } = await account.getTransactionHistoryV2({
-          orderBy: ExtrinsicsOrderBy.CreatedAtDesc,
+        const { data, count } = await account.getTransactionHistory({
+          orderBy: {
```

Comment:
@prashantasdeveloper is this the right update for the SDK. I was expecting the `ExtrinsicsOrderBy.CreatedAtDesc` field to remain for the v2 suffix removal.

I am not sure if BlockId is going to always order correctly because of string vs number ordering.

Reply:
As discussed on the call, I'm going to change this